ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -310,53 +310,6 @@ def getQueues(siteList=None, ceList=None, ceTypeList=None, community=None, mode=
   return S_OK(resultDict)
 
 
-def getStorageElements(vo=None):
```

Comment:
Why this function is removed ?

Reply:
Because it was never used and there's `DMSHelper` function.